tools/ci/run_flink_derived_feature_view_example.sh

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright 2022 The Feathub Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -e
+
+CURRENT_DIR=$(dirname "${BASH_SOURCE-$0}")
```

Comment:
Would it be simpler to add a script in the folder of each test that performs the test setup/teardown etc.?